bindings/node/native/src/classes/account_manager/mod.rs

```diff
@@ -298,7 +298,7 @@ declare_types! {
                 let guard = cx.lock();
                 let ref_ = &this.borrow(&guard).0;
                 let manager = ref_.read().unwrap();
-                manager.import_accounts(source).expect("error importing accounts");
+                crate::block_on(manager.import_accounts(source)).expect("error importing accounts");
```

Comment:
Are we sure we want to block here ?

Reply:
yeah this is fine for now but I'm planning on making this an async API in the future